en/mulesoft/modules/ROOT/pages/ms_composer_flows.adoc

```diff
@@ -1,10 +1,37 @@
-
 = Build a Composer Flow
 
 After you collect the information described in xref:ms_composer_checklist.adoc[the preparation checklist], you can then build, xref:ms_composer_activation.adoc[activate], and xref:ms_composer_monitoring.adoc[monitor] your flows. Building a flow involves creating a new flow or cloning an existing flow; adding and testing a trigger; adding and testing each action to be performed by the flow, in the order that you want each action performed; and finally testing the completed new flow before activating it.
 
 xref:ms_composer_test_flow.adoc[Test] each trigger and action as you add it to your flow.
 
+== Flow Details Page 
+
+Access the *Flow Details* page to see details about your flow:
+
+ifeval::["{product}"=="salesforce"]
+image::images/flow-details-sf.png[*Flow Details* page]
+endif::[]
+
+ifeval::["{product}"=="mulesoft"]
+image::images/flow-details-ms.png[*Flow Details* page]
+endif::[]
+
+[calloutlist]
+. The name of the flow. 
```

Comment:
This list should be an unordered list.

Reply:
Not in this situation - this is a callout list. The numbers in the list correspond with the numbers on image.